lib/rubocop/cop/bundler/gem_comment.rb

```diff
@@ -64,6 +76,8 @@ class GemComment < Base
         MSG = 'Missing gem description comment.'
         CHECKED_OPTIONS_CONFIG = 'OnlyFor'
         VERSION_SPECIFIERS_OPTION = 'version_specifiers'
+        LIMITING_VERSION_SPECIFIERS_OPTION = 'limiting_version_specifiers'
```

Comment:
I'm wondering if `restrictive`/`restricting` won't be better wording here.

Reply:
Agreed, I'm going to change it to "restrictive version".